src/app/public/src/modules/tutorial/tutorial-step.component.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../styles/_variables';
+
+/deep/
+.stache-tutorial-step {
+    border: 1px solid $sky-color-gray-lighter;
```

Comment:
Styles that do not require the `/deep/` label should be pulled out to prevent styles from leaking into the global scope.

```
.stache-tutorial-step {
  // Component-only styles...
@media () { ... }
}

/deep/
.stache-tutorial-step {
  .stache-row { }
}
```